adapters/adhese/adhese.go

```diff
@@ -18,267 +14,197 @@ import (
 	"github.com/prebid/prebid-server/v2/openrtb_ext"
 )
 
-type AdheseAdapter struct {
+type adapter struct {
 	endpointTemplate *template.Template
 }
 
-func extractSlotParameter(parameters openrtb_ext.ExtImpAdhese) string {
-	return fmt.Sprintf("/sl%s-%s", url.PathEscape(parameters.Location), url.PathEscape(parameters.Format))
+type ExtTargetsAdhese map[string][]string
+type ExtImpAdheseWrapper map[string]ExtTargetsAdhese
```

Comment:
`ExtTargetsAdhese ExtImpAdheseWrapper` type are accessible outside package `adhese`

scope of these types should be within package `adhese`

```
type extTargetsAdhese map[string][]string
type extImpAdheseWrapper map[string]ExtTargetsAdhese
```

Reply:
done!